### DISCUSSION TITLE
Registro en la plataforma

### DISCUSSION THREAD

**Discussion by C0SS10** [rramosp/ai4eng.v1#265] [UDEA Students]
Me he registrado con dos cuentas en la plataforma. Una con mi correo personal, sin saber que debía registrarme con el correo educativo. Hay alguna posibilidad de que se borre la cuenta creada con el correo personal (estebancogo47@gmail.com) y dejar la cuenta registrada con (esteban.cossio1@udea.edu.co). Gracias de antemano

**Comment by Felipe-RA** [comment]
Saludos Esteban

Recibido!
